### PR TITLE
add weights_format to package file name

### PR DESCRIPTION
**Pull request by FynnBe**
...if package is not containing all specified weights formats

### CODE REVIEW

**Review by constantinpape — APPROVED** [review]

**Comment by FynnBe** [comment]
waiting for approval of https://github.com/imjoy-team/bioimage-io-models/pull/5 to keep packagers in sync

**Comment by oeway** [comment]
You can go ahead and merge this now!

**Pull request merged**
